src/index.js

```diff
@@ -3,7 +3,7 @@
 const mh = require('multihashes')
 const multibase = require('multibase')
 const multicodec = require('multicodec')
-const { baseTable: codecs } = require('multicodec/src/base-table.js')
+const { nameToCode: codecs } = require('multicodec')
```

Comment:
Having `multicodec` imported twice is a bit strange, but I guess we want to move fast.